doc/whats_new/v0.22.rst

```diff
@@ -118,6 +118,9 @@ Changelog
   with a target matrix `Y` in which the first column was constant.
   :issue:`13609` by :user:`Camila Williamson <camilaagw>`.
 
+- |Feature| :class:`cross_decomposition._PLS` Has a new function :func:`cross_decomposition._PLS.inverse_transform` to transform 
+  data to the original space`. :pr:`15304` by :user:`Jaime Ferrando Huertas <jiwidi>`.
+
```

Comment:
Please keep the line length <80.

Also, we should point to the public API here, i.e. list the classes which inherit from `_PLS` instead.

Reply:
something like this maybe?

```
- |Feature| :class:`cross_decomposition.PLSCanonical` and 
  :class:`cross_decomposition.PLSRegression` 
  have a new function `inverse_transform` 
  :func:`cross_decomposition.PLSCanonical.inverse_transform`
  :func:`cross_decomposition.PLSRegression.inverse_transform` 
  to transform data to the original space`.
  :pr:`15304` by :user:`Jaime Ferrando Huertas <jiwidi>`.
```

Reply:
Yeah, I'd do something like:

```
- |Feature| :class:`cross_decomposition.PLSCanonical` and
  :class:`cross_decomposition.PLSRegression` have a new function
  ``inverse_transform`` to transform data to the original space`.
  :pr:`15304` by :user:`Jaime Ferrando Huertas <jiwidi>`.
```